packages/bpk-component-datepicker/readme.md

```diff
@@ -67,6 +67,8 @@ export default class App extends Component {
 > **Theming:** In order to theme the modal, a `renderTarget` needs to be supplied as a function which returns a DOM node
   in the scope of a `BpkThemeProvider`.
 
+> **getApplicationElement:** In order to be aligned to `microsites`, our examples use an element id value of `pagewrap`.
```

Comment:
How about:
`Note: the `pagewrap` element is a convention we use at Skyscanner which should "just work".`

I'd refrain from using terminology like "microsites".

Reply:
Cool. On it!